domain/identity/src/test/java/usecase/LogOutUseCaseTest.kt

```diff
@@ -0,0 +1,19 @@
+package usecase
+
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import repository.AuthenticationRepository
+import kotlin.test.Test
+
+class LogOutUseCaseTest {
+    private val authenticationRepository: AuthenticationRepository = mockk(relaxed = true)
+    private  val logoutUseCase: LogOutUseCase = LogOutUseCase(authenticationRepository)
+
+    @Test
+    fun `logout should call repository logout`() = runTest{
```

Comment:
It's not clear here what the repository is. I think you should name it
```suggestion
    fun `logout should call AuthenticationRepository when try to logout`() = runTest{
```